input/templates/pop.hbs

```diff
@@ -58,6 +58,9 @@
               <div class="jumbotron-row">
                 {{> catalog/jumbotron jumbotron=content.jumbotron}}
               </div>
+              {{!-- <div class="search-results-row">
+                {{> catalog/search-result search-result=content.searchResult}}
+              </div> --}}
               <div class="col-xs-4 hidden-xs text-left">
```

Comment:
It has to be included in a page that will show the search results, which is the same as the pop, except that instead of the `jumbotron`, we have this component.